### PR TITLE
Creating Journal-Week-2

### PR DESCRIPTION
**Pull request by JessyRiordan**

### CODE REVIEW

File: journal-week-2.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+# Week Two: 5/11 - 5/15
+
+I am only going to touch on the highlights of the week! :smile:
+
+## git status, git add, git commit, git push (still forget this one!) :smile:
+
+### Monday, May 11th
+
+* Starting the week off with a POP QUIZ... addin gsome of the quiz questions that I struggled with.
```
Comment (MStaehling) [inline journal-week-2.md]: >addin gsome

adding some

**Comment by MStaehling** [comment]
:+1:  Good outline.....gives me some ideas of what to add to mine

**Comment by pope410211** [comment]
:+1:  excellent outline, great start and cannot wait to read fleshed out version!

File: about/cascading-style-sheets.md [hunk 2]
```diff
@@ -0,0 +1,59 @@
+# CSS Layout Properties
+
+** This information has been taken from the [MDN](https://developer.mozilla.org/en-US/).** 
+
+## Bottom
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: Maybe use backticks? `bottom` 

Are CSS rules case-sensitive?

File: about/cascading-style-sheets.md [hunk 3]
```diff
@@ -0,0 +1,59 @@
+# CSS Layout Properties
+
+** This information has been taken from the [MDN](https://developer.mozilla.org/en-US/).** 
+
+## Bottom
+
+The bottom CSS property participates in specifying the position of positioned elements. 
+
+View the [MDN Reference on Bottom](https://developer.mozilla.org/en-US/docs/Web/CSS/bottom) for more information.
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: Maybe link the heading to the reference link instead:

> ## [`bottom`](https://developer.mozilla.org/en-US/docs/Web/CSS/bottom)

File: about/cascading-style-sheets.md [hunk 4]
```diff
@@ -0,0 +1,59 @@
...
+
+View the [MDN Reference on Right](https://developer.mozilla.org/en-US/docs/Web/CSS/right) for more information.
+
+## Top
+
+The top CSS property specifies part of the position of positioned elements. It has no effect on non-positioned elements.
+
+View the [MDN Reference on Top](https://developer.mozilla.org/en-US/docs/Web/CSS/top) for more information.
+
+## Visability
```
Comment (al-the-x) [inline about/cascading-style-sheets.md]: Is spelling important in CSS?

Comment (JessyRiordan) [inline about/cascading-style-sheets.md]: yes :(

**Comment by JessyRiordan** [comment]
@sjoyal I totally wrote that wrong... thanks for pointing it out! :)

**Comment by jorgehjr84** [comment]
Awesome recap of week 2 :+1:

**Comment by melrodbos** [comment]
Great job on your journal entry and also on making a list of the terminal commands we learned this week. I think we can all use the reference :+1: :smile:

**Comment by gatorpazz** [comment]
Nice CLI stuff, it's good to see a refresher of what we've done on a day to day basis from your journal, 10/10 would read again :+1:

**Comment by Epicurean306** [comment]
Great journal entry for this week. :+1: I like the way you organized things and the section on progress learning in Terminal is a good idea. :smile:

**Pull request merged**